desktop/main.dev.js

```diff
@@ -113,6 +113,8 @@ app.on('ready', async () => {
     mainWindow.webContents.send(ipcConsts.REQUEST_CLOSE);
   });
 
+  ipcMain.on(ipcConsts.CHECK_APP_VISIBLITY, () => mainWindow.webContents.send(ipcConsts.IS_APP_VISIBLE, { isAppVisible: mainWindow.isVisible() && mainWindow.isFocused() }));
```

Comment:
Sending object in response is meaningless in this case cause you do destruction and returns destructed value in response handler